ROMFS/px4fmu_common/init.d/rc.mc_apps

```diff
@@ -8,7 +8,7 @@
 # INAV, which defaults to 0 now. 
 if param compare INAV_ENABLED 1
 then
-	attitude_estimator_ekf start
+	attitude_estimator_q start
```

Comment:
What will make a difference is to reduce the coning errors by doing high-rate integration without filtering in the sensor driver - I've started to sketch this out here:
https://github.com/PX4/Firmware/pull/1975/files

Not ready / not compiling yet.